t3/utils/writer.py

```diff
@@ -204,13 +206,21 @@ def write_rmg_input_file(kwargs: dict,
     rmg_input += Template(model_template).render(**model)
 
     # simulator
-    simulator_template = """
-simulator(atol=${atol}, rtol=${rtol})
-"""
-    rmg_input += Template(simulator_template).render(atol=model_input['atol'], rtol=model_input['rtol'])
+    if t3['sensitivity']:
```

Comment:
I think we should check whether the T3 sensitivity adapter is RMG
Also, could `t3` be an empty dictionary or `None`? According to the schema it could. We might want to check for that first (i.e., check that it's not None, and check that 'sensitivity' is a key in this dict)

Reply:
Good point thank you. I updated the if statement to both check that `t3` is not `None` and that the `sensitivity` dictionary is present.